source/_faq/ha-vs-hassio.markdown

```diff
@@ -1,11 +1,13 @@
 ---
-title: "Home Assistant vs. Hass.io"
-description: "Home Assistant vs. Hass.io"
+title: "Home Assistant vs. Hass.io vs. HassOS"
+description: "Home Assistant vs. Hass.io vs. HassOS"
 ha_category: Installation
 ---
 
-Home Assistant is a Python program, in simple words. It can be run various operating system and provide the ability to track, control and automate your devices. When people talking about Home Assistant they usually refer to a standalone [installation method](/docs/installation/).
+Home Assistant is a Python program and is run via various [Installation Methods](https://www.home-assistant.io/docs/installation/).  The home assistant Python program is the core of the Home Assistant project that is used to observe, control and automate the devices in your home.  
```

Comment:
Besides adding a spelling error, I don't see any added value to this change.
I would suggest on reverting this part.